include/libdnf/repo/repo_callbacks.hpp

```diff
@@ -72,7 +72,9 @@ class RepoCallbacks {
     /// @param total_to_download total amount of bytes to download
     /// @param downloaded bytes downloaded
     /// @return TODO(lukash) this uses librepo's LrCbReturnCode, we should have our own enum
-    virtual int progress(double total_to_download, double downloaded) { return 0; }
+    virtual int progress(double total_to_download, double downloaded) {
+        return 0;
+    }
```

Comment:
I believe these are related to this clang-format bug https://github.com/llvm/llvm-project/issues/55260 (see also   https://github.com/rpm-software-management/ci-dnf-stack/pull/1110)
So I would not change this because we will need to change it back once the bug is resolved. The same for `handle_mirror_failure`, `get_base`, and bunch of methods in sqlite3/sqlite3.hpp